src/docdb/tree/DocDBDocumentTreeItem.ts

```diff
@@ -52,7 +55,7 @@ export class DocDBDocumentTreeItem implements IAzureTreeItem {
     }
 
     public async deleteTreeItem(_node: IAzureNode): Promise<void> {
-        const message: string = `Are you sure you want to delete document '${this.label}'?`;
+        const message: string = `Are you sure you want to delete document '${this._label}'?`;
```

Comment:
nit: Since you have a reader for "label"  you should probably still use "label" here.